### DISCUSSION TITLE
Obtained the same GND map for different EBSD data pack

### DISCUSSION THREAD

**Discussion by Qhgu** [mtex-toolbox/mtex#986] [Ask Anything] [answered]
Hi ralf,

I am sorry to bother you with my question on GND calculation. It is very strange that I always obtained the same GND map for different EBSD data. Please find the attached files.
I followed the instructions on these two pages:
https://mtex-toolbox.github.io/GND.html
https://mtex-toolbox.github.io/EBSDsquare.calcGND.html

The EBSD data are indexes of dual-phase titanium alloy, the primary phase α is HCP while the minor β phase is BCC for their atomic lattice, respectively.

I have made modification in the file @slipSystem slipSystem.m by adding the following lines under the "function sS = hcp(cs,varargin)"
>  sS = [slipSystem(Miller(-1,2,-1,0,cs,'uvtw'),Miller(0,0,0,1,cs,'hkil'),varargin{:}),...
>         slipSystem(Miller(2,-1,-1,0,cs,'uvtw'),Miller(0,1,-1,0,cs,'hkil'),varargin{:}),...
>         slipSystem(Miller(2,-1,-1,-3,cs,'uvtw'),Miller(1,0,-1,1,cs,'hkil'),varargin{:})];

I also have modified the lines for calculating the edge and screw energy to meet the case.
> nu = 0.34;
> Elastc = 110; % elastic modulus GPa
> G = Elastc/2/(1+nu); % shear modulus
> b = 1/3*a; % um
> dS(dS.isEdge).u = G*b^2/2;
> dS(dS.isScrew).u = G*b^2/2*(1 - nu);

MTEX 5.6.1 on MATLAB 2019a.

I can't come up with this issue and need your kind help.

Thanks and regards,
Gu

**Comment by ralfHielscher** [comment, marked as answer]
Hi Gu,

you could help us analyzing your issue by providing more information. The GND computation follows several steps.

1. computation of the incomplete curvature tensor kappa
2. computation of the incomplete Nye tensor alpha
3. fitting dislocation systems to the Nye tensor
4. computation of the energy

As you mention that you obtain the same result for different EBSD maps  the way to investigate the issue would be to follow this route:
1. have the same EBSD maps the same orientations?
2. do they have the same curvature tensors
3. do they have the same Nye tensors
4. ......

I think you got the idea. 

Ralf.

**Comment by Qhgu** [comment]
Hi Ralf,

All of the EBSD maps, oritentations, curvature tensors, and Nye tensors are different for these two EBSD data packs. I am very confused.

What further information can I provide to figure it out? 
The original files were attached in the main post. Can you check them?

Regards,
Gu

**Reply by ralfHielscher** [reply]
In the example script you attached you did after importing your data
```
mtexdata alphaBetaTitanium;
ebsd = ebsd('Ti (alpha)');
```
this overloads your data with the mtex example titanium data. Hence, the same result for all you input maps.

I wonder how you got different EBSD maps, oritentations, curvature tensors, and Nye tensors out of the same data file???

Ralf.

**Comment by Qhgu** [comment]
Hi Ralf,

I am terriblely sorry for my careless and the caused inconvenience.
Since it contains two phases, I use that two lines to restrict EBSD data to phase α under guidance of the tutorial, but with a mis-overload...

I quite don't catch the point to restrict the α phase in my case, can you give me any help?

Thank you very much!

Regards,
Gu

**Reply by ralfHielscher** [reply]
```matlab
ebsdAlpha = ebsd('Titanium (alpha)')
```

**Reply by Qhgu** [reply]
Well done Ralf. I have solved this issue with your valuable help.

Regard,
Gu
